tools/asciidoctor/generate-cli-docs

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+set -e
+set -u
+
+cd "$(dirname "${0}")"
+
+asciidoctor -b manpage -r ./postprocessor.rb -D ../../docs/ -o quick-lint-js.1 cli.adoc
+asciidoctor -b html5 -r ./postprocessor.rb -D ../../website/ -o cli.html cli.adoc
```

Comment:
Blocking: We should commit quick-lint-js.1 and cli.html into the repository. If we don't commit them, we'll require asciidoctor to be installed when building the Debian package, when installing with Nix, when building the manual packages (macOS and Linux), and when deploying the website. I don't want to require asciidoctor (and Ruby) for these workflows.